src/main/webapp/js/utils/table-cells/text-cell.jsx

```diff
@@ -1,21 +1,26 @@
+import _ from 'lodash';
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const TextCell = ({ value }) => (
+const TextCell = ({ format, pattern, value }) => (
   <div className="table-cell-text">
-    {value}
+    {_.isNil(format) && _.isNil(pattern) ? value : value.replace(pattern, format)}
```

Comment:
instead of `_.isNil, you can use `!format` and the same for pattern, it will work for empty string too, also change `&&` to `||`